hal/halx86/legacy/bus/pcibus.c

```diff
@@ -789,7 +792,7 @@ HalpAdjustPCIResourceList(IN PBUS_HANDLER BusHandler,
 #endif
 
     /* Return to caller */
-    ExFreePool(Interrupt);
+    ExFreePoolWithTag(Interrupt, TAG_HAL);
```

Comment:
Are we sure that the Interrupt buffer returned by the BusData->GetIrqRange is always allocated with TAG_HAL ?